### PR TITLE
Fs 139

### PR DESCRIPTION
**Pull request by ljgreenhill**
Commands for all SFRFields

### CODE REVIEW

File: src/sfr.cpp [hunk 1]
```diff
@@ -155,43 +155,43 @@ namespace sfr {
...
+        SFRField<uint16_t> attempts_limit = SFRField<uint16_t>(10, 0x1908);
     } // namespace burnwire
     namespace camera {
         // OP Codes 2000
-        SFRField<bool> photo_taken_sd_failed = SFRField<bool>(false, 2000);
-        SFRField<bool> take_photo = SFRField<bool>(false, 2001);
-        SFRField<bool> turn_on = SFRField<bool>(false, 2002);
-        SFRField<bool> turn_off = SFRField<bool>(false, 2003);
-        SFRField<bool> powered = SFRField<bool>(false, 2004);
+        SFRField<bool> photo_taken_sd_failed = SFRField<bool>(false, 0x2000);
```
Comment (Duncan-McD) [inline src/sfr.cpp:170]: do we want to go 0x1A00 if we are doing hex - (that way its sequential still) 

also this comment is applicable for rest below this as well

File: src/Control Tasks/RockblockControlTask.hpp [hunk 2]
```diff
@@ -39,13 +39,7 @@ class RockblockControlTask : public TimedControlTask<void>
     void dispatch_await_flush();
     void dispatch_end_transmission();
     void transition_to(rockblock_mode_type new_mode);
-    bool check_valid_command(RawRockblockCommand raw_command);
-    void check_timeout();
```
Comment (Duncan-McD) [inline src/Control Tasks/RockblockControlTask.hpp:43]: I see these two funcs didn't exist in the cpp file - were we planning to add them or are the not necessary

**Review by Duncan-McD — APPROVED** [review]
Approving because it looks good except for a couple clarifying comments

File: src/Control Tasks/RockblockControlTask.hpp (hunk 2 above)
Comment (ljgreenhill) [reply src/Control Tasks/RockblockControlTask.hpp:43]: They are not necessary

**Review by Duncan-McD — APPROVED** [review]
Looks good!

**Pull request merged**